urdf/test/test_robot_model_parser.cpp

```diff
@@ -42,92 +42,80 @@
 // args from command-line.
 #include <ros/ros.h>
 
-using namespace urdf;
-
 int g_argc;
-char** g_argv;
+char ** g_argv;
 
 class TestParser : public testing::Test
 {
 public:
-
   bool checkModel(urdf::Model & robot)
   {
     // get root link
     urdf::LinkConstSharedPtr root_link = robot.getRoot();
-    if (!root_link)
-    {
+    if (!root_link) {
       ROS_ERROR("no root link %s", robot.getName().c_str());
       return false;
     }
 
     // go through entire tree
     return this->traverse_tree(root_link);
-
-  };
+  }
 
 protected:
   /// constructor
   // num_links starts at 1 because traverse_tree doesn't count the root node
-  TestParser() : num_joints(0), num_links(1)
+  TestParser()
+  : num_joints(0), num_links(1)
   {
   }
 
-
   /// Destructor
   ~TestParser()
   {
   }
 
-  bool traverse_tree(urdf::LinkConstSharedPtr link,int level = 0)
+  bool traverse_tree(urdf::LinkConstSharedPtr link, int level = 0)
   {
     ROS_INFO("Traversing tree at level %d, link size %lu", level, link->child_links.size());
-    level+=2;
+    level += 2;
     bool retval = true;
-    for (std::vector<urdf::LinkSharedPtr>::const_iterator child = link->child_links.begin(); child != link->child_links.end(); child++)
+    for (auto child = link->child_links.begin();
```

Comment:
this is C++ 11 and cannot be used upstream before maybe melodic. So it should be moved to another commit

Reply:
this still needs to be addressed

Reply:
Oops, missed it.  Fixed now.